Support for linux/riscv64

currently there're such errors below:
```
# github.com/refraction-networking/utls/cpu
src/github.com/refraction-networking/utls/cpu/cpu.go:16:16: undefined: CacheLineSize
src/github.com/refraction-networking/utls/cpu/cpu.go:33:16: undefined: CacheLineSize
src/github.com/refraction-networking/utls/cpu/cpu.go:46:14: undefined: CacheLineSize
src/github.com/refraction-networking/utls/cpu/cpu.go:58:14: undefined: CacheLineSize
src/github.com/refraction-networking/utls/cpu/cpu.go:66:14: undefined: CacheLineSize
src/github.com/refraction-networking/utls/cpu/cpu.go:76:14: undefined: CacheLineSize
```

--- a/cpu/cpu_riscv64.go
+++ b/cpu/cpu_riscv64.go
@@ -1,0 +1,11 @@
+// Copyright 2019 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build riscv64
+
+package cpu
+
+const cacheLineSize = 32
+
+func doinit() {}

